setup.py

```diff
@@ -3,7 +3,7 @@
 
 # To rebuild just the cython modules, use these commands:
 # sudo apt-get install python-setuptools cython
-# python setup.py build_ext --inplace
+# python3 setup.py build_ext --inplace
 from __future__ import division, print_function
```

Comment:
With the upgrade from python2 --> python3 these import can maybe be removed?

Reply:
Indeed. I'll make a separate PR with removal of all these imports, for all the files.